MRN-Code/coinstac
# InputMap Data Prop Refactor, Parsing InputMap Covar Types, File Group Delete

# Problem
* Closes #465 
* Closes #459 
* Closes #456 
* Closes #423 
* Closes #424 
* Closes #474 
* Covariate columns when mapping local data are not filtered by type
* Substrings in columns aren't auto-selected on contains
* Remote pipeline errors not displaying in UI
* ipcRenderer listener errors if refreshed too many times
# Solution
* Removing file groups now causes mapped associated consortia that utilize them to reset
* Owner defined covar types now determine how covariate items are parsed
* Data property in `inputMap` only one level deep
* Display property with `type: table` of comp spec more fleshed out to work with multiple regressions
   * Pipeline step output now works with nested objects
   * Results display nested tables
* Computation output included with results for label access
* Run details displayed with their results, including covars if present
* Covariate columns now look at type definition to filter out incorrect types
* Looks for substring matches in auto-column selecting
* `error` now included in GraphQL query to retrieve remote errors in UI
* Remove listeners on `local-run-error` and `local-pipeline-state-update` in `dashboard` unmount call
# Testing
* Map a new file group to a consortium to see covar filtering
* Delete the file group used to map above and see that the mapping is cleared
* Run `ssr_test` using multiple ROIs and view results
* InputMap refactoring shouldn't have UI impact, though the parsed values can be seen in the console after starting a pipeline
